crates/api/src/site.rs

```diff
@@ -197,7 +202,7 @@ impl Perform for Search {
     let search_type: SearchType = from_opt_str_to_opt_enum(&data.type_).unwrap_or(SearchType::All);
     let community_id = data.community_id;
     let community_actor_id = if let Some(name) = &data.community_name {
-      resolve_actor_identifier::<Community>(name, context.pool())
+      resolve_actor_identifier::<ApubCommunity, Community>(name, context)
```

Comment:
Wasnt able to make this work with only one generic param. Anyway the type checks make sure you cant mess it up, and its only used in few places.